proofs/salts_test.go

```diff
@@ -0,0 +1,51 @@
+package proofs
+
+import (
+	"github.com/centrifuge/precise-proofs/examples/documents"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestFillSalts(t *testing.T) {
```

Comment:
I think it would be nicer to separate out the cases in this test into multiple tests